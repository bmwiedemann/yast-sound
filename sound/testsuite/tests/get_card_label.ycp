{
    include "testsuite.ycp";
    map READ_I = $[
	"target" : $[
	    "size": -1,
	    "tmpdir" : "/tmp"
	]
    ];

    TESTSUITE_INIT ([READ_I], nil);
    include "sound/routines.ycp";

    map card = $[];

    TEST(``(get_card_label(card)), [], nil);

    card = $["bus":"PCI", "class_id":4, "device":"SB Live! EMU10000", "device_id":65538, "drivers":[$["active":false, "modprobe":true, "modules":[["emu10k1", ""]]]], "resource":$["io":[$["active":true, "length":0, "mode":"rw", "start":49152]], "irq":[$["count":1146364, "enabled":true, "irq":10]]], "rev":"7", "sub_class_id":1, "sub_device":"CT4830 SBLive! Value", "sub_device_id":98342, "sub_vendor":"Creative Labs", "sub_vendor_id":69890, "unique_key":"CvwD.k1dGGUobcK9", "vendor":"Creative Labs", "vendor_id":69890];

    TEST(``(get_card_label(card)), [], nil);

    card = $["bus":"PCI", "class_id":4, "device":"", "device_id":65538, "drivers":[$["active":false, "modprobe":true, "modules":[["emu10k1", ""]]]], "resource":$["io":[$["active":true, "length":0, "mode":"rw", "start":49152]], "irq":[$["count":1146364, "enabled":true, "irq":10]]], "rev":"7", "sub_class_id":1, "sub_device":"CT4830 SBLive! Value", "sub_device_id":98342, "sub_vendor":"Creative Labs", "sub_vendor_id":69890, "unique_key":"CvwD.k1dGGUobcK9", "vendor":"Creative Labs", "vendor_id":69890];

    TEST(``(get_card_label(card)), [], nil);
    
    card = $["bus":"PCI", "class_id":4, "device":"", "device_id":65538, "drivers":[$["active":false, "modprobe":true, "modules":[["emu10k1", ""]]]], "resource":$["io":[$["active":true, "length":0, "mode":"rw", "start":49152]], "irq":[$["count":1146364, "enabled":true, "irq":10]]], "rev":"7", "sub_class_id":1, "sub_device":"", "sub_device_id":98342, "sub_vendor":"Creative Labs", "sub_vendor_id":69890, "unique_key":"CvwD.k1dGGUobcK9", "vendor":"Creative Labs", "vendor_id":69890];

    TEST(``(get_card_label(card)), [], nil);
    
    card = $["bus":"PCI", "class_id":4, "device":"", "device_id":65538, "drivers":[$["active":false, "modprobe":true, "modules":[["emu10k1", ""]]]], "resource":$["io":[$["active":true, "length":0, "mode":"rw", "start":49152]], "irq":[$["count":1146364, "enabled":true, "irq":10]]], "rev":"7", "sub_class_id":1, "sub_device":"CT4830 SBLive! Value", "sub_device_id":98342, "sub_vendor":"Creative Labs", "sub_vendor_id":69890, "unique_key":"CvwD.k1dGGUobcK9", "vendor":"Creative Labs", "vendor_id":69890];

    TEST(``(get_card_label(card)), [], nil);

}
