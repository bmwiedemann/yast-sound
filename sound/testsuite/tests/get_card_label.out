Read	.target.tmpdir "/tmp"
Read	.target.size "/etc/install.inf" -1
Read	.probe.architecture nil
Read	.probe.has_apm nil
Read	.probe.has_pcmcia nil
Read	.probe.has_smp nil
Read	.probe.system nil
Return	
Return	CT4830 SBLive! Value
Return	CT4830 SBLive! Value
Return	Creative Labs
Return	CT4830 SBLive! Value
